src/components/evaluation/EvaluationTable.vue

```diff
@@ -316,14 +307,14 @@ export default {
       'ignore': {label: 'Ignore', enabled: false}
     },
     headers: [
-      {text: 'Status', value: 'status'},
-      {text: 'Last Updated', value: 'lastUpdated'},
-      {text: 'Course Number', value: 'sortableCourseNumber', width: '90px'},
-      {text: 'Course Name', value: 'sortableCourseName', width: '200px'},
-      {text: 'Instructor', value: 'sortableInstructor'},
-      {text: 'Department Form', value: 'departmentForm.name', width: '170px'},
-      {text: 'Evaluation Type', value: 'evaluationType.name', width: '100px'},
-      {text: 'Evaluation Period', value: 'startDate', width: '200px'}
+      {class: 'text-nowrap', text: 'Status', value: 'status'},
+      {class: 'text-nowrap', text: 'Last Updated', value: 'lastUpdated'},
+      {class: 'text-nowrap', text: 'Course Number', value: 'sortableCourseNumber', width: '90px'},
+      {class: 'text-nowrap', text: 'Course Name', value: 'sortableCourseName', width: '200px'},
+      {class: 'text-nowrap', text: 'Instructor', value: 'sortableInstructor'},
+      {class: 'text-nowrap', text: 'Department Form', value: 'departmentForm.name', width: '170px'},
+      {class: 'text-nowrap', text: 'Evaluation Type', value: 'evaluationType.name', width: '100px'},
+      {class: 'text-nowrap', text: 'Evaluation Period', value: 'evaluationPeriod.start', width: '200px'}
```

Comment:
@lyttam This might have crossed paths with the recent evaluation-date PR. I think we do now want `startDate` for this value, not `evaluationPeriod.start`.

Reply:
@pauline2k good catch, I overlooked that in the merge.